lib/options.js

```diff
@@ -12,7 +12,15 @@ const DEFAULTS = {
     // Should the plugin handle the select all inside a code container
     selectAll: true,
     // Allow marks inside code blocks
-    allowMarks: false
+    allowMarks: false,
+    // Custom exit handler
+    onExit: (transform, options) => {
+        // Exit the code block
+        transform.insertBlock({ type: options.exitBlockType });
```

Comment:
We should mention that `exitBlockType` option is useless if `onExit` is provided